workloads/artful.yaml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Could you split out the new workloads into separate commits?  One commit for both this and the fedora27 workload would be fine.